app/views/govuk_publishing_components/components/_emergency_banner.html.erb

```diff
@@ -7,6 +7,7 @@
   link_text ||= "More information"
   campaign_class ||= nil
   homepage ||= false
+  ga4_tracking ||= false
```

Comment:
Presumably there'll be a separate PR in `static` to set this to `true`?

Reply:
Yep, I'll do this as part of `To deploy` on the card.